src/literature/components/NumberOfPages.jsx

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+class NumberOfPages extends Component {
+  render() {
+    const { numberOfPages } = this.props;
+
+    if (!numberOfPages) {
+      return null;
+    }
+
+    return <span>{numberOfPages} pages</span>;
```

Comment:
if there is only one page?

Reply:
I think we need to decide what way to go and do it like that everywhere, page(s) or you want to have conditional suffix everywhere?

Reply:
conditional suffix everywhere to be consistent, cc @StellaCh

Reply:
I agree